cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -110,6 +111,14 @@ def test_verify_revert_snapshot(test_vm, provider, soft_assert, register_event,
         'hostname': ip
     }
     with SSHClient(**ssh_kwargs) as ssh_client:
+        # We need to wait for ssh to become available on the vm, it can take a while. Without
+        # this wait, the ssh command would fail with 'port 22 not available' error.
+        # Easiest way to solve this is just mask the exception with 'handle_exception = True'
+        # and wait for successful completition of the ssh command.
+        # The 'fail_func' ensures we close the connection that failed with exception.
+        # Without this, the connection would hang there and wait_for would fail with timeout.
+        wait_for(lambda: ssh_client.run_command('touch snapshot1.txt').rc == 0, num_sec=300,
+                 delay=20, handle_exception=True, fail_func=ssh.close())
         ssh_client.run_command('touch snapshot1.txt')
```

Comment:
With the previous change, you technically do not need to run this command since the successful execution of the wait_for lambda will do exactly the same.